src/meta/proto-conv/tests/it/proto_conv.rs

```diff
@@ -333,5 +336,29 @@ fn test_build_pb_buf() -> anyhow::Result<()> {
         println!("copied_file_lock:{:?}", buf);
     }
 
+    // schema
+    {
+        let b1 = TableDataType::Tuple {
+            fields_name: vec!["b11".to_string(), "b12".to_string()],
+            fields_type: vec![TableDataType::Boolean, TableDataType::String],
+        };
+        let b = TableDataType::Tuple {
+            fields_name: vec!["b1".to_string(), "b2".to_string()],
+            fields_type: vec![b1, TableDataType::Number(NumberDataType::Int64)],
+        };
+        let fields = vec![
+            TableField::new("a", TableDataType::Number(NumberDataType::UInt64)),
+            TableField::new("b", b),
+            TableField::new("c", TableDataType::Number(NumberDataType::UInt64)),
+        ];
+        let schema = TableSchema::new(fields);
+
+        let p = schema.to_pb()?;
+
+        let mut buf = vec![];
+        common_protos::prost::Message::encode(&p, &mut buf)?;
+        println!("schema:{:?}", buf);
+    }
```

Comment:
This test is a duplicate of 
[v026_schema.rs](https://github.com/datafuselabs/databend/pull/9623/files#diff-ee4eeb4f2627c5f0629a9c4b0a07f4d5f231b5d9da8be3008fed9c400db9e9ad) and can be removed.